src/Api/index.js

```diff
@@ -169,14 +183,14 @@ class Api {
 }
 
 function handleFetch(maybeUrl = '', options = {}) {
-  const url = replace(maybeUrl, /^\//g, '')
+  const [, version, url] = /^(?:([a-z]+):)?\/?(.*)$/i.exec(maybeUrl)
```

Comment:
So to call a Helix endpoint, we append `helix:` to the URL.

What about using `options` to determine the endpoint version? For example:
```js
api.get('streams?game_id=33214', { apiVersion: 'helix' })
````

Reply:
Let's do it one way ... less to document 😆

Reply:
So, which one? Yours?

Reply:
Well, both are understandable, but each API is different so the option one could mean it can possible to switch between them.

I also remembered Symfony, the PHP framework, use namespaces in their view paths with the following format, `@namespace/path`. It'd be possible to use instead of the current proposal. Which means the new way will be, for calling an Helix endpoint, `@helix/endpoint`.

Or, maybe creating a sub-API object for each version (`api.kraken.get(...)`, `api.helix.put(...)`) could be also possible, but requires documentation.